src/app/detail/detail.html

```diff
@@ -30,6 +30,13 @@
         <hr/>
 
         <a href data-ng-click="detail.loadJobsWithCategory(detail.job.publishedCategory.id)" class="category-filter">{{'detail.viewAllJobsLabel' | i18n}}<i class="bhi-arrow-right"></i></a>
+        <div class='ios-upload-warning' data-ng-if="detail.isIOS">
+            <div>{{'detail.iosUploadWarning' | i18n}}</div>
+        </div>
+    </div>
+    <div class='send' data-ng-if="detail.isIOS">
+        <input class="email" type="text" data-ng-model="detail.email" id="email" name="email" data-i18n-attr="{placeholder: 'common.emailPlaceholder'}" placeholder="{{common.emailPlaceholder  | i18n}}">
```

Comment:
The `data-i18n-attr` should take care of the `placeholder` attribute, don't need both.